src/components/onboarding/Onboarding.tsx

```diff
@@ -13,9 +21,9 @@ const POPUP_HEIGHT = 140;
  *
  * Displays guided onboarding tooltips around selected UI elements.
  */
-const Onboarding: React.FC = () => {
+const Onboarding: React.FC<OnboardingProps> = ({ screen }) => {
```

Comment:
```suggestion
const Onboarding = ({ screen }: OnboardingProps) => { ... }
```
Recommended not to use React.FC any longer, and I can see you were just adding props to the line that was already there but this is as good a time as any to refactor it

Reply:
Replaced React.FC as suggested.